holidays/locale/th/LC_MESSAGES/CN.po

```diff
@@ -59,3 +68,20 @@ msgstr "วันเด็กสากล"
 #. Army Day.
 msgid "建军节"
 msgstr "วันสถาปนากองทัพปลดปล่อยประชาชนจีน"
+
+#. Date format (see strftime() Format Codes).
+msgid "%Y-%m-%d"
+msgstr "%d/%m/%Y"
+
+#. Day off (substituted from %s).
+#, c-format
+msgid "休息日（%s日起取代）"
+msgstr "วันพัก (แทนที่จาก %s)"
+
+#. 70th Anniversary of the Victory of the Chinese People’s War of Resistance
+#. against Japanese
+#. Aggression and the World Anti-Fascist War.
+msgid "中国人民抗日战争暨世界反法西斯战争胜利70周年纪念日"
+msgstr ""
+"ครบรอบ 70 "
```

Comment:
[The official Thai translation](https://oldweb.ditp.go.th/contents_attach/132578/132578.pdf) uses "ครบรอบ 70 ปีแห่งการได้รับชัยชนะจากสงครามต่อต้านญี่ปุ่นและลัทธิฟาสซิสต์โลก" here so we might want to go with that instead.

(they got some typo in the referenced translation above for the Thai translation of "Fascism" - corrected to [Royal Society of Thailand](https://dictionary.orst.go.th/)'s official spelling)